docs/ramalama-login.1.md

```diff
@@ -49,9 +49,11 @@ Login to huggingface registry
 ```
 $ ramalama login --token=XYZ huggingface
 ```
+Logging in to Hugging Face requires the `huggingface-cli` tool. For installation and usage instructions, see the [documentation of Command Line Interface (CLI) of Hub Python Library on Hugging Face].
```

Comment:
Logging in to Hugging Face requires the `huggingface-cli` tool. For installation and usage instructions, see the [*documentation of Command Line Interface (CLI) of Hub Python Library on Hugging Face*].